corehq/apps/export/urls.py

```diff
@@ -10,7 +10,7 @@
     poll_custom_export_download,
     prepare_custom_export,
     prepare_form_multimedia,
-)
+    DownloadDETSchemaView)
```

Comment:
nit: isort/formatting
```suggestion
    DownloadDETSchemaView,
)
```
(I cannot suggest proper sorting here because github does not allow me to select enough lines)